app/views/register/company/directors/whatYouWillNeed.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import viewmodels.Message
+
+@(appConfig: FrontendAppConfig, href: Call)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("whatYouWillNeed.directors.title"),
+    appConfig = appConfig,
+    bodyClasses = None
+) {
+
+    @components.heading(headingKey = messages("whatYouWillNeed.directors.heading"))
+
+    <p>@messages("whatYouWillNeed.directors.body.text")</p>
+
+    <ul class="list list-bullet">
```

Comment:
all these messages can be reused for partners, hence it would be better if they are given a generic name